ogc/docs/conf.md

```diff
@@ -0,0 +1,350 @@
+# OGC Server Configuration
```

Comment:
Is there an example of an entire sample configuration file?

ie...
something like a  `sample_config.json` in the docs?